### ISSUE TITLE
folder and file list does not showup in 0.3.0 and 0.3.1 version. However, works fine in with 0.2.0 version

### ISSUE THREAD

**Issue by shek2994** [crahan/ipyfilechooser#25]
my setup details:
jupyter lab version:- 1.1.4
jupyter nbextension:- 6.0.1

ipyfilechooser version = 0.3.1
```from ipyfilechooser import FileChooser

# Create and display a FileChooser widget
fc = FileChooser('/home/som/Downloads/pytest_dir')
display(fc)
```
screenshot of the widget output below
[https://drive.google.com/file/d/1HwYjAEma27bEXr8X4VB4ezqxeReWKSjr/view?usp=sharing](url)

ipyfilechooser version 0.2.0 (screenshot of working widget)
[https://drive.google.com/file/d/1I_aysgWrVArA_GsRMAL7AuWfrxbQ9yLK/view?usp=sharing](url)

**Comment by crahan** [comment]
Thanks for reporting. I'm not sure what might be causing this, but for reference my Jupyter lab and nbextension versions are as follows (tested on macOS 10.15.5):

Jupyter lab: 2.2.0
Jupyter nbextension: 6.0.3

Can you try with the latest 0.3.4 version of ipyfilechooser?

**Comment by crahan** [comment]
I'd to follow up on this issue. Were you able to test with the more recent 0.3.4 or (just released) 0.3.5 versions?

**Comment by crahan** [comment]
Hi @shek2994, version v0.4.0 has just been released to PyPi and conda-forge. If you have some time, can you check if this issue persists?
